tests/integration/utils/kiali_client.go

```diff
@@ -26,6 +29,12 @@ type AuthStrategy struct {
 
 var client = *NewKialiClient()
 
+var BOOKINFO = "bookinfo"
+var ASSETS = "tests/integration/assets"
+var VS = path.Join(cmd.KialiProjectRoot, ASSETS+"/bookinfo-reviews-80-20.yaml")
+var DR = path.Join(cmd.KialiProjectRoot, ASSETS+"/bookinfo-destination-rule-reviews.yaml")
+var WORKLOADS = path.Join(cmd.KialiProjectRoot, ASSETS+"/bookinfo-workloads.yaml")
```

Comment:
Are these specific to a single test or shared across multiple? If they are specific to a single test then it'd be better to create them inside that test. Otherwise we'll end up with a bunch of these as global vars.

Reply:
+1